### PR TITLE
Stack.c

### PR DESCRIPTION
**Pull request by Rishav-00**
How to Write a Stack Code in C

## Have you read the Contribution Guidelines?

(Write your answer here.)

## Description

(Write your answer here.)

## Checklist
- [x] I've read the contribution guidelines.
- [ ] I've checked the issue list before deciding what to submit.
- [ ] I've edited the README.md and link to my code.
 
 
## Related Issues (Mention Issue no. like #xxx)

Fixes Issue 
<br/>
(Write your answer above.)

### CODE REVIEW

**Comment by sukritishah15** [comment]
@Rishav-00 - Please mention the Issue # it fixes in the PR description.

**Review by sukritishah15 — CHANGES_REQUESTED** [review]
@Rishav-00 
Please name the file properly as stack.c if you're implementing it in C.

File: Implementation of Stack [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+   while(!isempty()) {
+      int data = pop();
+      printf("%d\n",data);
+   }
+
+   printf("Stack full: %s\n" , isfull()?"true":"false");
+   printf("Stack empty: %s\n" , isempty()?"true":"false");
+   
+   return 0;
+}
```
Comment (sukritishah15) [inline Implementation of Stack:71]: Please add Sample Input/Output and Time and Space Complexity at the end of the file in a comment.
Thank you!

Comment (Rishav-00) [reply Implementation of Stack:71]: Ok

**Comment by Rishav-00** [comment]
now please accept my pull request

File: Implementation of Stack (hunk 1 above)
Comment (Rishav-00) [reply Implementation of Stack:71]: now please accept my pull request

**Comment by sunilgknair051** [comment]
@Rishav-00 , fair warning to read the contributing.md, and please make sure you answer the questions when commenting.

**Comment by sukritishah15** [comment]
@Rishav-00 - Please make the requested changes.

**Pull request closed without merging**